etc/grub.d/19_linux_xen_trenchboot

```diff
@@ -147,7 +148,7 @@ EOF
       message="$(gettext_printf "Loading SINIT module %s ..." ${sinit_module})"
       sed "s/^/$submenu_indentation/" << EOF
 	echo	'$message'
-	module	/${sinit_module}
+	slaunch_module	/${sinit_module}
```

Comment:
slaunch_module has been dropped possibly?

Reply:
No, it is still there.